### PR TITLE
Simple dev, bare minimum for new features

### PR DESCRIPTION
**Pull request by KonoganB**
Allows for an easier management of the new projects

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -345,6 +345,8 @@ app.post('/project/:id/request', projects);
 app.delete('/project/:id', projects);
 app.put('/project/:id/request', projects);
 app.post('/ask/project', projects);
+app.get('/pending', projects);
```
Comment (KonoganB) [inline app.js:348]: New routes

File: manager2/src/app/admin/projects/projects.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
   <div *ngIf="notification" class="alert alert-success">{{notification}}</div>
   <div class="card bg-light">
     <div class="card-header" (click)="requests_visible=!requests_visible" style="cursor: pointer;">
-      <h3>{{requests_number}} request(s) waiting </h3>
+      <h3>{{requests_number + pending_number}} request(s) waiting </h3>
```
Comment (KonoganB) [inline manager2/src/app/admin/projects/projects.component.html:4]: separated the number of requests from the number of pending projects. Underneath that is the added UI elements allowing their management

File: manager2/src/app/admin/projects/projects.component.ts [hunk 3]
```diff
@@ -270,4 +306,28 @@ export class ProjectsComponent implements OnInit {
         }
         return res;
     }
+
+    accept_project(project) {
```
Comment (KonoganB) [inline manager2/src/app/admin/projects/projects.component.ts:310]: 3 methods to take a decision for the pending project:
accept -> straight to backend, no path or additional info added. to be filled later from the existing project management page.
modify -> fills this.new project and admin can use the legacy way of accepting the project, and can modify or fill in the missing info.
reject: straight to backend, pending project is rejected

File: manager2/src/app/admin/projects/projects.component.ts [hunk 4]
```diff
@@ -220,6 +230,8 @@ export class ProjectsComponent implements OnInit {
                                        this.add_project_error_msg = err.error.message;
                                    }
                                );
+            
+            console.log("hello")
```
Comment (osallou) [inline manager2/src/app/admin/projects/projects.component.ts:234]: debug log to remove

File: app.js (hunk 1 above)
Comment (osallou) [reply app.js:348]: wouldbe better to get "named" routes:

/pending/project , /pending/project/:id

best to handle later other pending stuff

File: tests/docker-compose.yml [hunk 5]
```diff
@@ -82,6 +82,7 @@ services:
             - MY_ADMIN_GROUP=admin
             - MY_ADMIN_EMAIL=${MY_ADMIN_EMAIL:-admin@my.org}
             - MY_ADMIN_PASSWORD=${MY_ADMIN_PASSWORD:-admin}
+            - gomngr_auth=fake
```
Comment (osallou) [inline tests/docker-compose.yml:85]: nope this is for your own debug

File: routes/utils.js [hunk 6]
```diff
@@ -37,6 +38,7 @@ var mongo_connect = async function() {
     mongo_web = mongodb.collection('web');
     mongo_projects = mongodb.collection('projects');
     mongo_tags = mongodb.collection('tags');
+    mongo_pending = mongodb.collection('pending');
```
Comment (osallou) [inline routes/utils.js:41]: should name table pending_projects or alike

File: routes/utils.js [hunk 7]
```diff
@@ -37,6 +38,7 @@ var mongo_connect = async function() {
     mongo_web = mongodb.collection('web');
     mongo_projects = mongodb.collection('projects');
     mongo_tags = mongodb.collection('tags');
+    mongo_pending_projects = mongodb.collection('pending');
```
Comment (osallou) [inline routes/utils.js:41]: mongo collection name should be pending_projects

File: manager2/src/app/admin/projects/projects.service.ts [hunk 8]
```diff
@@ -178,4 +178,44 @@ export class ProjectsService {
...
+        let params = new HttpParams();
+
+        let httpOptions = {
+            //headers: new HttpHeaders({
+            //  'x-api-key': user.apikey
+            //}),
+            params: params
+        };
+        return this.http.delete(
+            environment.apiUrl + '/pending/project' + projectId,
```
Comment (osallou) [inline manager2/src/app/admin/projects/projects.service.ts:216]: missing / I think

**Pull request merged**